tests/data/itunes_keywords.json

```diff
@@ -0,0 +1,5 @@
+{
+	"title": "itunes_keywords",
+	"episodes": [],
+	"itunes_keywords": "adventure, tabletop, rpg"
```

Comment:
This should be parsed into a list of strings instead? Might be more useful? Especially since the docs mention "whitespace is squashed", it'd be kind of pointless if just leading and trailing whitespace is squashed, but not whitespace between elements.

```json
    "itunes_keywords": ["adventure", "tabletop", "rpg"]
```

Reply:
maybe instead make both options available? my worry is that there might be a weirdly formatted keyword that doesnt deliminate correctly like this, or that for some reason they wont be deliminated by commas

Reply:
Does the specification allow that? Podcastparser strictly follows the spec and doesn't bend for broken feeds.

Reply:
Yeah, I'd go for enforcing it to be keywords, which is probably what iTunes and the Podcasts app does.

For other reasons why, see the "Criticism" section in Postel's law Wikipedia page: https://en.wikipedia.org/wiki/Robustness_principle#Criticism

While trying to find the spec for the keywords, I stumbled upon this one:
https://discussions.apple.com/thread/8232215

So most likely, the official Apple apps ignore the keywords (probably because it was used for SEO or something).

So I'd be for one of:

- Implement it strictly (list-of-strings keywords, parsed as comma-separated list)
- Drop keyword support, as it's not used officially by Apple (the "owners" of the XML namespace that's usually aliased to `itunes:`)

Reply:
id lean on still including it, and ill just implement the comma list parsing instead